terms.html

```diff
@@ -22,47 +22,48 @@
-  <li>“<em>content</em>” means files (including scripts) or data which you upload to or create using <em>the platform</em></li>
+  <li>“<em>content</em>” means files which you upload to or create using PDF Tables</li>
   <li>“intellectual property” means any rights in patents, registered and unregistered designs, copyright, databases, trade marks, confidential information and all other forms of intellectual property wherever in the world enforceable.</li>
   <li>“<em>the parties</em>” means us and you.</li>
-  <li>“<em>the pricing plan</em>” is the particular set of service features and associated charging level which you most recently agreed to.</li>
+  <li>“<em>the pricing plan</em>” is the particular level of use and associated charges which you most recently agreed to.</li>
 </ul>
-<p>Where a defined term is used, it will be written in italic for clarity.</p>
+<p>Where a defined term is used, it will be written in italic script for clarity.</p>
 <p><span class="section-number">3</span>The terms “<em>writing</em>”, “<em>written</em>” etc include any document in electronically readable form, and in particular, electronic mail, but do not include ephemeral systems of communication such as twitter or text messaging.</p>
 <p><span class="section-number">4</span>Where you are required to give us notice under <em>this agreement</em>, this must be done by means of direct communication such as a website form specifically provided for the purpose of contacting us, a letter, or an email.  For the avoidance of doubt, notice by means of a wiki, technical support system, internet forum or other similar mechanism shall not be valid notice under <em>this agreement</em>.</p>
 <p><span class="section-number">5</span>We may choose to annotate <em>this agreement</em> to explain why we have chosen to draft it in this way. If we do so, those annotations will not form a part of this contract or have contractual force.</p>
 <h2>Our commitment to you</h2>
-<p><span class="section-number">6</span>We will provide access to <em>the platform</em> as detailed in <em>the pricing plan</em>, and subject to the limitations (bandwidth, storage, etc.) contained in that plan.  We will exercise reasonable care and skill in providing <em>the platform</em>, and will do what is reasonable in the circumstances to ensure that <em>the platform</em> is operational as far as is possible.</p>
+<p><span class="section-number">6</span>We will provide access to PDF Tables as detailed in <em>the pricing plan</em>, and subject to the limitations (eg. as to file size or number of fiels)contained in that plan.  </p>
+<p>PDF's can be complicated files. It is impossible for us to guarantee that the tables extracted from a PDF will be accurate. All we promise to do is exercise reasonable skill and care in maintaining PDF Tables as a service that works reasonably well.</p>
```

Comment:
Line 40 ("fiels").

Line 41: Is this a promise we can live up to, and does it have any contractual basis? "works reasonably well" sounds like quite a strong statement to me; for some users, it won't work at all (because they uploaded a PDF which is missing critical information). The form of the breakage might not be obvious at all, for example a number in a conversion could be subtly incorrect, or everything might be completely garbled. I could imagine someone suggesting in both of these cases that "it doesn't work reasonably well".